src/main/java/net/snowflake/client/core/CredentialManager.java

```diff
@@ -106,16 +116,26 @@ synchronized void fillCachedCredential(SFLoginInput loginInput, String credType)
     }
 
     if (cred == null) {
-      logger.debug("retrieved %s is null", credType);
+      logger.debug("Retrieved %s is null", credType);
     }
 
     // cred can be null
     if (credType == ID_TOKEN) {
+      logger.debug(
+          "Setting {}id token for user: {}, host: {}",
+          cred == null ? "null " : "",
+          loginInput.getUserName(),
+          loginInput.getHostFromServerUrl());
       loginInput.setIdToken(cred);
     } else if (credType == MFA_TOKEN) {
+      logger.debug(
+          "Setting {}mfa token for user: {}, host: {}",
+          cred == null ? "null " : "",
+          loginInput.getUserName(),
+          loginInput.getHostFromServerUrl());
       loginInput.setMfaToken(cred);
     } else {
-      logger.debug("unrecognized type %s for local cached credential", credType);
+      logger.debug("Unrecognized type %s for local cached credential", credType);
```

Comment:
let's replace %s with {} in the log message

Reply:
Done